src/resource/resource.ts

```diff
@@ -165,6 +166,13 @@ export function internal_parseResourceInfo(
         resourceInfo.sourceIri
       ).href;
     }
+    // Parse all link headers and expose them in a standard way
+    // (this can replace the parsing of the ACL link above):
+    const linkedResources = parsedLinks.refs.reduce((rels, ref) => {
+      rels[ref.rel] = new URL(ref.uri, resourceInfo.sourceIri).href;
+      return rels;
+    }, {} as Record<string, string>);
```

Comment:
Here, I could see `linkedResources` be `{}` if `linkHeader` is undefined (related to my comment above).